### DISCUSSION TITLE
Add Multiple (Secondary) Time Picker

### DISCUSSION THREAD

**Discussion by mertbakir** [grafana/grafana#38228] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

I need a second time picker where I can select an alternative time range beside the primary one. We should be able to use it with the existing macros like this:
```
$__timeFilter2("foo")
$__timeFrom2()
$__timeTo2()
```

The secondary time picker should be disabled as default and can be enabled in the dashboard settings.

**Why is this needed**:

I am working on a dashboard which compares two selected time periods. So, I have created a variable and use it as 'date_offset' over the selected time period but this has it's limitation:

* It's complicated for an avg. dashboard user.
* The second period is always dependent on the first period. (Always same time intervals, only offsetting)
* Using a variable doesn't look good.
* Only limited options will be available in the offset variable (1 day, 7 day, 1 month etc.)

_Sorry, if this has already requested but I couldn't find a similar issue._

**Comment by hugohaggmark** [comment]
@torkelo I remember you mentioning something similar to this last week, do you have an existing issue for that work?

**Comment by dprokop** [comment]
Would it make sense to have additional time range control in the context of a single query @torkelo @hugohaggmark? If set, it would override the dashboard's set time range. I think @peterholmberg is working on sth like this in the alerting project.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
